node/router/messages/src/lib.rs

```diff
@@ -111,8 +112,13 @@ impl<N: Network> From<DisconnectReason> for Message<N> {
 }
 
 impl<N: Network> Message<N> {
-    /// The version of the network protocol; it can be incremented in order to force users to update.
-    pub const VERSION: u32 = 17;
+    /// The version of the network protocol; this can is incremented for breaking changes between migration versions.
+    pub const VERSIONS: [(ConsensusVersion, u32); 2] = [(ConsensusVersion::V4, 16), (ConsensusVersion::V5, 17)];
```

Comment:
Cool! Can you add relevant tests on these constants similar to `test_consensus_constants` in snarkVM?

Reply:
Added in [3eca0df](https://github.com/ProvableHQ/snarkOS/pull/3596/commits/3eca0dfa4df035e58f66ebc482c3bcc94dd3119a)